docs/rhoas/cli-context/README.adoc

```diff
@@ -0,0 +1,240 @@
+== Prerequisites
+
+[role="_abstract"]
+* You've installed the `rhoas` CLI. For more information, see {base-url}{rhoas-cli-installation-url}[_Installing the rhoas CLI_^].
+
+[id="proc-creating-context-cli_{context}"]
+== Creating a service context
+
+.Procedure
+
+Enter the following command to create a context named `my-context`
+
+.Creating a context
+[source,shell]
+----
+$ rhoas context create --name my-context
+----
+
+[NOTE]
+====
+By default, the created context is set as the current context.
+All the service based commands are executed against the service instances set in the current context.
+====
+
+[id="proc-setting-services-cli_{context}"]
+== Setting services in a context
+
+[role="_abstract"]
+Context contains identifiers to service instances.
+All the service based commands are executed against the service instances set in the current context.
+
+.Procedure
+
+. Setting Kafka Instance in the current context
++
+--
+.Set Kafka instance in the current context
+[source,shell]
+----
+$ rhoas context set-kafka
+----
+--
+
+. Setting Service Registry Instance in the current context
++
+--
+.Set Service Registry instance in the current context
+[source,shell]
+----
+$ rhoas context set-service-registry
+----
+
+[NOTE]
+====
+Creating a new service instance automatically sets it in the current context.
+====
+--
+
+. Check status of the current context
++
+--
+After setting service(s) in the current context, their staus can be checked using the `rhoas context status` command
+.View status of the current context
+[source,shell]
+----
+$ rhoas context status
+----
+--
+
+[id="proc-generating-config-cli_{context}"]
+== Generating application configurations for contexts
+
+`rhoas generate-config` command generates application configurations that can be used to connect to the services set in context.
+
+.Generating configurations for context named as my "my-context"
+[source,shell]
+----
+$ rhoas generate-config --name my-context --type json
+----
+
+=== Generating configurations for quarkus applications
+
+[id="proc-importing-quarkus-sample-code_{context}"]
+==== Importing the Quarkus sample code
+
+[role="_abstract"]
+For this quick start, you'll use the Quarkus sample code from the {product-kafka} {samples-git-repo}[Guides and Samples^] repository in GitHub.
+After you understand the concepts and tasks in this quick start, you can use your own Quarkus applications with {product-kafka} in the same way.
+
+.Procedure
+. On the command line, clone the {product-kafka} {samples-git-repo}[Guides and Samples^] repository from GitHub.
++
+.Cloning the guides and samples repository
+[source,subs="+attributes"]
+----
+git clone {samples-git-repo} app-services-guides
+----
+. In your IDE, open the `code-examples/quarkus-kafka-quickstart` directory from the repository that you cloned.
+. Generate the application configurations for the quarkus application.
++
+--
+[source,shell]
+----
+$ rhoas generate-config --type env --output-file .env 
```

Comment:
Any reason we repeat previous step?

Reply:
This command is specific to the quarkus example, the command above is just an usage example with default values mostly.